src/Microsoft.Health.Fhir.Core/Resources.resx

```diff
@@ -175,8 +175,35 @@
     <value>Found result with Id '{0}', which did not match the provided Id '{1}'.</value>
     <comment>{0} is an Id e.g. 123456</comment>
   </data>
-  <data name="CustomerManagedKeyInaccessible" xml:space="preserve">
-    <value>The customer managed key is not accessible.</value>
+  <data name="CmkAadClientCredentialsGrantFailure" xml:space="preserve">
+    <value>AAD failure getting a token to access the customer managed key.</value>
+  </data>
+  <data name="CmkAadServiceUnavailable" xml:space="preserve">
+    <value>AAD service unavailable to get access to the customer managed key.</value>
+  </data>
+  <data name="CmkKeyVaultAuthenticationFailure" xml:space="preserve">
+    <value>The customer managed key is disabled, or the Key Vault does not grant permission to AAD.</value>
+  </data>
+  <data name="CmkKeyVaultKeyNotFound" xml:space="preserve">
+    <value>The customer managed key is not found.</value>
+  </data>
```

Comment:
Grammar:  I think customer-managed key is correct.  Also, when I look at the BYOK documentation guidance, a '-' is used.  I will switch to that.